docs/whirlpool/package.json

```diff
@@ -5,11 +5,14 @@
   "scripts": {
     "build": "docusaurus build --out-dir dist",
     "start": "docusaurus start",
+    "swizzle": "docusaurus swizzle",
```

Comment:
What does this command do (and do we really need it as a script in the package.json)?

Reply:
It can create copies of components in your own repo, so you can add styling to them. Not really needed to have it as a script.

Reply:
If not really needed can we remove?